README.md

```diff
@@ -29,18 +29,20 @@ npm install --save fh-mbaas-api
 
 This will install the latest version of fh-mbaas-api and save the installed version in your package.json
```

Comment:
@evanshortiss not sure what do u mean by this? Do you mean the code to init the new sync?